NeoML/docs/en/API/NN/TiedEmbeddingsLayer.md

```diff
@@ -0,0 +1,42 @@
+# CTiedEmbeddingsLayerd Class
+
+<!-- TOC -->
+
+- [CTiedEmbeddingsLayerd Class](#ctiedembeddingslayerd-class)
+  - [Settings](#settings)
+    - [EmbeddingsLayerName](#embeddingslayername)
+    - [ChannelIndex](#channelindex)
+  - [Inputs](#inputs)
+  - [Outputs](#outputs)
+
+<!-- /TOC -->
+
+Class implements tied embeddings layer.

```

Comment:
Also link to the article is needed here. [Example](https://github.com/neoml-lib/neoml/blob/05168e5233d7dd9ba40acdec4a0095dc05cce96c/NeoML/docs/en/API/NN/LossLayers/CenterLossLayer.md)